pkg/backends/templates/templates.go

```diff
@@ -0,0 +1,71 @@
+package templates
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/trickstercache/trickster/v2/pkg/backends/options"
+	ustrings "github.com/trickstercache/trickster/v2/pkg/util/strings"
+)
+
+var templateBackends map[string]*options.Options = make(map[string]*options.Options)
```

Comment:
how does one reset the templateBackends back to an empty map (e.g., when reloading the application config when it has changed)?